spec/behavior/auth.spec.js

```diff
@@ -4,7 +4,7 @@ var HyperResource = require( "../../src/hyperResource.js" );
 
 var board1 = model.board1;
 
-describe( "Authorization", function() {
+describe.only( "Authorization", function() {
```

Comment:
That's awkward... :)


On Jan 28, 2016, 7:36 PM -0600, Alex Robson<notifications@github.com>, wrote:
>  
> Inspec/behavior/auth.spec.js(https://github.com/LeanKit-Labs/hyped/pull/27#discussion_r51215773):
>  
> >@@ -4,7 +4,7 @@ var HyperResource = require( "../../src/hyperResource.js" );>>var board1 = model.board1;>>-describe( "Authorization", function() {>+describe.only( "Authorization", function() {
>  
> @dcneiner(https://github.com/dcneiner)- oops :)
>  
>  
> —
> Reply to this email directly orview it on GitHub(https://github.com/LeanKit-Labs/hyped/pull/27/files#r51215773).
>  
>  
>